libsolidity/codegen/Compiler.h

```diff
@@ -59,14 +59,14 @@ class Compiler
 	/// @returns Only the runtime object (without constructor).
 	eth::LinkerObject runtimeObject() const { return m_context.assembledRuntimeObject(m_runtimeSub); }
 	/// @arg _sourceCodes is the map of input files to source code strings
-	std::string assemblyString(StringMap const& _sourceCodes = StringMap()) const
+	std::string assemblyString(StringMap _sourceCodes = StringMap()) const
```

Comment:
These functions here should not be changed, the parameters are not handed over, they are just referenced.